### ISSUE TITLE
Kubectl shouldn't print throttling debug output 

### ISSUE THREAD

**Issue by bprashanth** [kubernetes/kubernetes#23007]
This might be fixed in newer clusters:
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"3+", GitVersion:"v1.3.0-alpha.0.377+80f33256a2f9e1-dirty", GitCommit:"80f33256a2f9e16403d78dad6780bf5f381ee566", GitTreeState:"dirty"}
Server Version: version.Info{Major:"1", Minor:"3+", GitVersion:"v1.3.0-alpha.0.164+712b14aa86f2f0-dirty", GitCommit:"712b14aa86f2f03233c5f81bf8c2c846734d272a", GitTreeState:"dirty"}

$ kubectl delete svc,rc --all
service "echoheadersdefault" deleted
service "echoheadersx" deleted
service "echoheadersy" deleted
service "haproxy" deleted
service "hostname" deleted
service "kubernetes" deleted
W0315 13:43:08.562657   29271 request.go:627] Throttling request took 101.029465ms, request: GET:https://104.197.205.46/api/v1/namespaces/default/replicationcontrollers/hostname
W0315 13:43:08.762632   29271 request.go:627] Throttling request took 157.263869ms, request: GET:https://104.197.205.46/api/v1/namespaces/default/replicationcontrollers
replicationcontroller "hostname" deleted
replicationcontroller "service-loadbalancer" deleted
```

**Comment by AdoHe** [comment]
@bprashanth because we use `glog.Warningf` here, maybe we should update the log level.

**Comment by AdoHe** [comment]
This is not fixed in the v1.2.0 #23107, we should update this.

**Comment by liggitt** [comment]
Definitely should be lower verbosity, and the `longThrottleLatency` of 50ms is questionable as well, given that the default client QPS is 5

**Comment by krmayankk** [comment]
What is remaining here ? it seems this is already fixed in #23175 or else i can try sending this as my first PR
